pkg/apis/componentconfig/validation/validation_pluginargs.go

```diff
@@ -102,3 +109,80 @@ func ValidateRemoveDuplicatesArgs(args *componentconfig.RemoveDuplicatesArgs) er
 		validateNamespaceArgs(args.Namespaces),
 	)
 }
+
+// validateThresholds checks if thresholds have valid resource name and resource percentage configured
+func validateThresholds(thresholds api.ResourceThresholds) error {
```

Comment:
nit: I prefer having private methods below all public methods

Reply:
hi, since validateThresholds is only used by like `ValidateHighNodeUtilizationArgs`, so i keep it private :)

Reply:
I'm not asking to change method visibility.

I am asking to re-order as follows:

```
// all public methods on top of the file

// ValidateHighNodeUtilizationArgs ...
func ValidateHighNodeUtilizationArgs(...)

// all private methods at the bottom of the file
func validateNamespaceArgs(...)
```

As you currently have it, there's private methods and public all mixed together. Anyways, not a big deal, just a formatting nit.

Reply:
sure, i will update now